### PR TITLE
screen preview

### PR DESCRIPTION
**Pull request by awinogradov**

### CODE REVIEW

**Comment by awinogradov** [comment]
cc @Yeti-or

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@ static
 dmg
 !bundles/**/*.bemjson.js
 .enb/tmp/
+Super Dynamics Project/
```
Comment (Yeti-or) [inline .gitignore]: wtf?

File: src/lib/compilers/compilers.tests/index.js [hunk 2]
```diff
@@ -20,15 +20,19 @@ describe('Compilers', () => {
       name: 'screen1',
       cwd: join(projectPath, 'screen1'),
       projectCwd: projectPath,
+      markup: '<svg><g>text</g></svg>',
       scripts: 'console.log(1)'
     })._getState();
 
-    mock({[projectPath]: mock.directory()});
+    // FIXME: SVGO doesn't work with mock-fs
```
Comment (Yeti-or) [inline src/lib/compilers/compilers.tests/index.js]: how about issue?

File: src/lib/compilers/index.js [hunk 3]
```diff
@@ -31,3 +34,35 @@ exports.scripts = function(screen) {
...
+    {removeViewBox: true},
+    {collapseGroups: false},
+    {removeHiddenElems: false},
+    {cleanupIDs: false}
+  ]});
+
+  return Promise
+    .resolve()
+    .then(() => {
+      svgo.optimize(screen.markup, optimized => {
```
Comment (Yeti-or) [inline src/lib/compilers/index.js]: could you explain why we need svgo here?

File: src/protein-tree/project/index.js [hunk 4]
```diff
@@ -8,17 +8,16 @@ const createLogger = require('redux-logger-server');
...
     rootReducer,
     restoredProject,
-    applyMiddleware(
-      thunkMiddleware,
-      promiseMiddleware(),
-      createLogger({
-        color: 'green'
-      })
-    )
+    applyMiddleware.apply(applyMiddleware, middlewares)
```
Comment (Yeti-or) [inline src/protein-tree/project/index.js]: why not `applyMiddleware(middlewares)`  ? or I miss smth?

File: src/lib/compilers/compilers.tests/index.js (hunk 2 above)
Comment (awinogradov) [inline src/lib/compilers/compilers.tests/index.js]: we need to remove svgo

File: src/lib/compilers/index.js (hunk 3 above)
Comment (awinogradov) [inline src/lib/compilers/index.js]: no benefits

File: src/protein-tree/project/index.js (hunk 4 above)
Comment (awinogradov) [inline src/protein-tree/project/index.js]: code in the top of file ;)

**Pull request merged**